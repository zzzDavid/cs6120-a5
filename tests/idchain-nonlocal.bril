# CMD: bril2json < {filename} | python ../main.py -const_prop | python ../dce.py -g | bril2txt
@main {
  x: int = const 4;
  jmp .label;
.label:
  copy1: int = id x;
  copy2: int = id copy1;
  copy3: int = id copy2;
  print copy3;
}
