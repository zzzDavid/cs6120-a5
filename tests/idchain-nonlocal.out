@main {
.start:
  x: int = const 4;
  jmp .label;
.label:
  print x;
}
